docs/guides/usage.rst

```diff
@@ -30,42 +30,61 @@ General Usage
 
 The above command allows the user to infer metadata for single- or paired-ended RNA-Seq libraries by specifying file paths and relevant parameters. The tool outputs metadata in JSON format to :code:`STDOUT` and logs to :code:`STDERR`.
 
-Command-line Options
+Command-line options
 ---------------------
 
 Available command-line parameters are categorized as follows:
 
-- **General Options**:
-    - :code:`--output-directory`: Path where output data will be saved.
-    - :code:`--temporary-directory`: Path for storing temporary files generated during execution.
-    - :code:`--cleanup-regime`: Specifies which data should be kept after completion. Available options are :code:`DEFAULT`, :code:`KEEP_ALL`, :code:`KEEP_NONE`, and :code:`KEEP_RESULTS`.
-    - :code:`--verbosity`: Controls the verbosity level of log output; options are :code:`DEBUG`, :code:`INFO`, :code:`WARN`, :code:`ERROR`, and :code:`CRITICAL`.
+.. list-table:: General Options
```

Comment:
Can you somehow left-align the table header? If not, can you remove it and put some regular text (maybe in bold) above each table?

Reply:
Apparently it's not possible, added the headers instead.